### DISCUSSION TITLE
Refresh layout to have the Reviewer banner when you assign yourself

### DISCUSSION THREAD

**Discussion by kud** [community/community#37619] [Pull Requests]
**Describe the bug**

When you set yourself as a reviewer, you need to refresh the whole page to get the reviewer banner.

**To Reproduce**

- Go to a PR
- Add yourself as reviewer

**Expected behaviour**

I should see this banner:

![image](https://user-images.githubusercontent.com/655838/198565983-5ea5a7a3-fd23-4882-b5f9-3dbcccbee3e3.png)

You got this banner only when you refresh the page.

**Comment by ossanna16** [comment]
Hi there @kud and welcome to our community! Can you please elaborate on your question/issue further?

**Reply by kud** [reply]
Hey @ossanna16.  Is this clearer?

**Reply by ossanna16** [reply]
@kud Yes a lot, thank you 🙂
